src/views/Profile/ProfileNftCard.js

```diff
@@ -0,0 +1,206 @@
+          }}
+        />
+        <ToniqButton
+          text="Transfer"
+          onClick={event => {
+            props.onTransferClick({
+              id: userNft.token,
+              listing: userNft.listing,
+            });
+            event.stopPropagation();
+          }}
+        />
+      </>
+    ) : props.currentTab === ProfileTabs.Activity ? (
+      <div style={{display: 'flex', flexDirection: 'column'}}>
+        <span
+          style={{
+            ...cssToReactStyleObject(toniqFontStyles.boldParagraphFont),
+          }}
+          title={formattedDateString}
+        >
+          {relativeDate}
+        </span>
+        <span
+          style={{
+            ...cssToReactStyleObject(toniqFontStyles.labelFont),
+            color: String(toniqColors.pageSecondary.foregroundColor),
+          }}
+        >
+          {userNft.statuses.has(nftStatusesByTab[ProfileTabs.Activity].Bought) ? 'Bought' : 'Sold'}
+        </span>
+      </div>
+    ) : props.currentTab === ProfileTabs.Watching ? (
+      <>
+        <div style={{display: 'flex', flexDirection: 'column'}}>
+          <span
+            style={{
+              ...cssToReactStyleObject(toniqFontStyles.boldParagraphFont),
+              alignSelf: 'flex-end',
+            }}
+          >{`${userNft.offers.length} Offer${userNft.offers.length === 1 ? '' : 's'}`}</span>
+          <span
+            style={{
+              ...cssToReactStyleObject(toniqFontStyles.labelFont),
+              color: String(toniqColors.pageSecondary.foregroundColor),
+            }}
+          >
+            {includesYourOffer ? 'including yours' : ''}
+          </span>
+        </div>
+      </>
+    ) : (
+      <></>
+    );
+
+  return isListView ? (
+    <>
+      <style
+        dangerouslySetInnerHTML={{
+          __html: String(css`
+            .profile-list-view-nft-details-row > span {
+            }
+          `),
+        }}
+      ></style>
+      <ListRow
+        items={[
+          '',
+          `#${userNft.mintNumber}`,
+          userNft.nri ? (userNft.nri * 100).toFixed(1) + '%' : '-',
+          listing,
+          extraDetails,
+        ]}
+      />
+    </>
+  ) : (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+      }}
+    >
+      <span
+        style={{
+          marginBottom: '16px',
+          marginTop: '16px',
+          ...cssToReactStyleObject(toniqFontStyles.h3Font),
+        }}
+      >
+        #{userNft.mintNumber}
+      </span>
+      <div style={{display: 'flex'}}>
+        <div
+          style={{
+            flexGrow: 1,
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '8px',
+          }}
+        >
+          <span style={cssToReactStyleObject(toniqFontStyles.boldParagraphFont)}>{listing}</span>
+          <span style={cssToReactStyleObject(toniqFontStyles.labelFont)}>
+            {userNft.nri ? `NRI: ${userNft.nri * 100}%` : ''}
```

Comment:
```suggestion
{userNft.nri ? `NRI: ${(userNft.nri * 100).toFixed(1)}%` : ''}
```

Some of the card NRI has long values
![Screen Shot 2022-09-27 at 10 55 14 AM](https://user-images.githubusercontent.com/99699764/192509091-3a960d9a-9828-4236-b400-cedc4fd39fa5.png)